alfresco-transform-imagemagick/alfresco-transform-imagemagick-boot/pom.xml

```diff
@@ -81,6 +81,14 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <systemPropertyVariables>
+                        <IMAGEMAGICK_EXE>/usr/bin/convert</IMAGEMAGICK_EXE>
+                        <IMAGEMAGICK_DYN>/usr/lib64/ImageMagick-7.0.7/lib</IMAGEMAGICK_DYN>
+                        <IMAGEMAGICK_ROOT>/usr/lib64/ImageMagick-7.0.7</IMAGEMAGICK_ROOT>
```

Comment:
Should we configure 7.0.7 as a top-level pom version property to help we catch references (at least poms & Dockerfiles) .... we would still need to remember to also update defaults in ootb YAML files.

Reply:
These property variables are only used in tests as they are part of the surefire-plugin.

Reply:
OK, that's fine - we need to remember to change in all places when we next upgrade ImageMagick version.